src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12402,7 +12408,7 @@ namespace UnitarySystems {
                                 //                   RegulaFalsi may not find latent PLR when the latent degradation model is used.
                                 //                   IF iteration limit is exceeded, find tighter boundary of solution and repeat RegulaFalsi
                                 TempMaxPLR = -0.1;
-                                TempOutletHumRatDXCoil = OutletHumRatDXCoil;
+                                TempOutletHumRatDXCoil = DesOutHumRat;
```

Comment:
This was causing SolveRoot failure = -2 because a tighter min/max limit of 0.01 in the SolveRoot call was too tight (e.g., min=0.78, max=0.79).